packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/InAppPurchasePlugin.swift

```diff
@@ -0,0 +1,412 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Foundation
+import StoreKit
+
+#if os(iOS)
+  import Flutter
+#elseif os(macOS)
+  import FlutterMacOS
+#endif
+
+extension FlutterError: Error {}
+@objcMembers
+public class InAppPurchasePlugin: NSObject, FlutterPlugin, InAppPurchaseAPI {
+  // Properties
+  private(set) var productsCache: NSMutableDictionary = [:]
+  private(set) var paymentQueueDelegateCallbackChannel: FlutterMethodChannel?
+  private(set) var receiptManager: FIAPReceiptManager?
+  private(set) var paymentQueueDelegate: Any? = nil
+  // note - the type should be FIAPPaymentQueueDelegate, but this is only available >= iOS 13,
+  private var requestHandlers = Set<FIAPRequestHandler>()
+  private var handlerFactory: ((SKRequest) -> FIAPRequestHandler)?
+  public var registrar: FlutterPluginRegistrar?
+  public var paymentQueueHandler: FIAPaymentQueueHandler?
+  public var transactionObserverCallbackChannel: FlutterMethodChannel?
```

Comment:
> these shouldn't be public. public means it can be called from outside of this module.
For tests - the objc test file can't seem to find them if theyre only private

Reply:
maybe im missing something here - it's marked as resolved, but the code is not updated.

Reply:
wtf i swear i added a comment

Reply:
Is this not the comment? (https://github.com/flutter/packages/blob/b300519b60588d257abd6a655172d82e999875e3/packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/InAppPurchasePlugin.swift#L29)

Reply:
Can you put the comment before the code?

Reply:
Discussed offline - I didn't see the comment previously because the comment is after the code, so it looks like a comment for the `register` function